File: quality_test_face/compute_kB_image.py
from PIL import Image
import os

img = Image.open("/home/maicg/Documents/Me/CERBERUS/CerberusAISDK/database/face_25_4/person60/find_person_person60_0.510000_24.484400.jpg")
file_size = os.path.getsize("/home/maicg/Documents/Me/CERBERUS/CerberusAISDK/database/face_25_4/person60/find_person_person60_0.510000_24.484400.jpg") / 1024 # chuyển đổi kích thước sang kB
print("Dung lượng của ảnh là", file_size, "kB")

File: quality_test_face/compute_kB_image.cpp
#include <iostream>
#include <opencv2/opencv.hpp>
#include <filesystem>

using namespace std;
using namespace cv;
namespace fs = std::filesystem;

int main() {
    Mat img = imread("path/to/image.jpg");
    int file_size = fs::file_size("path/to/image.jpg") / 1024; // chuyển đổi kích thước sang kB
    cout << "Dung lượng của ảnh là " << file_size << " kB" << endl;
    return 0;
}